### ISSUE TITLE
An error occurred while Style/EmptyElse cop

### ISSUE THREAD

**Issue by bobmaerten** [rubocop/rubocop#13187]
Since 1.66, I get errors from Style/EmptyElse with plain simple if statements.

```ruby
    if paymethod.not?(:card)
      tag.div(logo)
    end
```

## Expected behavior

There should not be any runtime errors

## Actual behavior

```
An error occurred while Style/EmptyElse cop was inspecting <file>
```

## Steps to reproduce the problem

Run `rubocop` on codebase of a simple file contaning an if statement

## RuboCop version

```
$ [bundle exec] rubocop -V
1.66.0 (using Prism 1.0.0, rubocop-ast 1.32.2, running on ruby 3.3.4) [arm64-darwin23]
```

Same behavior with native Parser 3.3.4.2

**Comment by koic** [comment]
This issue is a dup with #13176 and resolved in #13177. Thank you for the feedback.
